hardhat.config.ts

```diff
@@ -29,7 +29,7 @@ export default {
       },
     },
     tenderly: {
-      chainId: vars.has("TENDERLY_CHAIN_ID") ? vars.get("TENDERLY_CHAIN_ID") : 1,
+      chainId: vars.has("TENDERLY_CHAIN_ID") ? parseInt(vars.get("TENDERLY_CHAIN_ID")) : 1,
```

Comment:
Be explicit about the parsing base:
```suggestion
      chainId: vars.has("TENDERLY_CHAIN_ID") ? parseInt(vars.get("TENDERLY_CHAIN_ID"), 10) : 1,
```